Classes/VTAcknowledgementsViewController.m

```diff
@@ -394,6 +396,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     VTAcknowledgement *acknowledgement = self.acknowledgements[indexPath.row];
     cell.textLabel.text = acknowledgement.title;
     cell.accessoryType  = UITableViewCellAccessoryDisclosureIndicator;
+	cell.textLabel.textColor = self.textColor;
+	cell.backgroundColor = self.cellBackgroundColor;
+	UIView *selectedBackgroundView = self.cellSelectedBackgroundColor ? [[UIView alloc] init] : nil;
```

Comment:
@moritzmoritz Happy to use `new` if you prefer. My sources imply preferring alloc/init as the standard, however: https://twitter.com/bbum/status/834105958259253248